tensorflow/compiler/tf2tensorrt/convert/utils.h

```diff
@@ -73,6 +78,31 @@ struct VectorTensorShapeHasher {
    (NV_TENSORRT_MAJOR == major && NV_TENSORRT_MINOR == minor && \
     NV_TENSORRT_PATCH == patch && NV_TENSORRT_BUILD >= build))
 
+// This utility template converts an arithmetic type to a string. This function
+// is necessary to allow the following function to behave recursively:
+// `string DebugString(const std::vector<CType>&)`.
+template<
+    typename CType,
+    typename = typename std::enable_if<
+      std::is_arithmetic<CType>::value, CType
+    >::type
+>
+string DebugString(const CType& el) {
+  string el_str = std::to_string(el);
+  // This lines prettifies std::to_string which can sometimes returns
```

Comment:
We can remove "This lines" or replace it with "This line".

Reply:
Sorry, need to mention if you will remove "This lines", you also need to s/prettifies/Prettify/

Reply:
Fair enough. Fixed